packages/data-stores/src/site/actions.ts

```diff
@@ -438,7 +438,7 @@ export function createActions( clientCreds: WpcomClientCredentials ) {
 		selectedDesign: Design,
 		options?: DesignOptions
```

Comment:
I wonder if we can remove `verticalId` from the `DesignOptions` type because it is only used in this file and we are removing it.